### DISCUSSION TITLE
NvChad install directory.

### DISCUSSION THREAD

**Discussion by MrHeer** [NvChad/NvChad#1091] [Q&A] [answered]
Is there any way to install NvChad's configuration file in other directory? I just want to keep the custom directory in my nvim config directory.

**Comment by amirhhashemi** [comment, marked as answer]
No, you can't. At least not easily.

However, you may find something useful here: https://github.com/NvChad/NvChad/issues/1014

**Reply by MrHeer** [reply]
Is it possible to publish NvChad as a plug-in, and then let us customize NvChad like a plug-in?

**Reply by amirhhashemi** [reply]
maybe you can but why? NvChad is already clean and minimal. Is it really a problem to have a few more files in your neovim config directory?

**Reply by MrHeer** [reply]
I want to completely separate my custom configuration from NvChad's configuration, so that I can put my custom files in my git repository instead of tracking the NvChad

**Reply by MrHeer** [reply]
Focus separation, I just need to pay attention to my custom part. NvChad updates are like updating a dependency package.

**Reply by siduck** [reply]
@MrHeer ask about this in our discord/matrix server, someone could help you. I heard an answer regarding this before, i cant recall it

**Reply by siduck** [reply]
also you might need to have to look at the default configs which is present in nvchad so putting nvchad into another dir can bring you some extra work

**Reply by MrHeer** [reply]
Thank you so much.
